src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -1,67 +1,133 @@
-        rearLeftSpark = new CANSparkMax(RobotConstants.CAN.REAR_LEFT_MOTOR, MotorType.kBrushless);
-        leftMotorController = new MotorControllerGroup(frontLeftSpark, rearLeftSpark);
-
-        frontRightSpark = new CANSparkMax(RobotConstants.CAN.FRONT_RIGHT_MOTOR, MotorType.kBrushless);
-        rearRightSpark = new CANSparkMax(RobotConstants.CAN.REAR_RIGHT_MOTOR, MotorType.kBrushless);
-        rightMotorController = new MotorControllerGroup(frontRightSpark, rearRightSpark);
-        frontLeftEncoder = frontLeftSpark.getEncoder();
-        frontRightEncoder = frontRightSpark.getEncoder();
-        rearLeftEncoder = rearLeftSpark.getEncoder();
-        rearRightEncoder = rearRightSpark.getEncoder();
-        robotDrive = new DifferentialDrive(leftMotorController, rightMotorController);
-    }
+  private boolean useFieldAbsolute = false;
 
-    @Override
-    public void initSendable(SendableBuilder builder) {
-        builder.setSmartDashboardType(getName());
-        builder.addDoubleProperty("FL Encoder Ticks", () -> { return frontLeftEncoder.getPosition(); }, null);
-        builder.addDoubleProperty("FR Encoder Ticks", () -> { return frontRightEncoder.getPosition(); }, null);
-        builder.addDoubleProperty("RL Encoder Ticks", () -> { return rearLeftEncoder.getPosition(); }, null);
-        builder.addDoubleProperty("RR Encoder Ticks", () -> { return rearRightEncoder.getPosition(); }, null);
-    }
+  private NavXSubSystem navX; 
 
-    public void feedWatchDog(){
-        robotDrive.feed();
-    }
+  private DriveInput loggingDriveInput = new DriveInput(0, 0, 0);
+  
+  /**
+   *
+   * 
+   * @param NavX The NavXSubsystem that some filters use and the drive in field absolute
+   */
+  public DriveSubsystem(NavXSubSystem NavX) {
+    navX = NavX;
+  }
+
+  public DrivePose getDriveOutput(){
+      return new DrivePose();
+  }
+  
+  @Override
+  public void initialize() {
+    frontLeftTalon  = new WPI_TalonSRX(RobotConstants.CAN.FRONT_LEFT_MOTOR);
+    rearLeftTalon   = new WPI_TalonSRX(RobotConstants.CAN.REAR_LEFT_MOTOR);
+    frontRightTalon = new WPI_TalonSRX(RobotConstants.CAN.FRONT_RIGHT_MOTOR);
+    rearRightTalon  = new WPI_TalonSRX(RobotConstants.CAN.REAR_RIGHT_MOTOR);
+    robotDrive      = new MecanumDrive(frontLeftTalon, rearLeftTalon, frontRightTalon, rearRightTalon);
+    DFM             = new DriveFilterManager(navX);
+
+    robotDrive.setDeadband(0.1);
+
+    frontLeftTalon.setInverted(false);
+    rearLeftTalon.setInverted(false);
+    frontRightTalon.setInverted(true);
+    rearRightTalon.setInverted(true);
+    
+    frontLeftTalon.enableCurrentLimit(false);
+    rearLeftTalon.enableCurrentLimit(false);
+    frontRightTalon.enableCurrentLimit(false);
+    rearRightTalon.enableCurrentLimit(false);
+  }
+
+  @Override
+  public void periodic() {
 
-    @Override
-    public void periodic() {
-        feedWatchDog();
+    SmartDashboard.putNumber("Front Left Talon", frontLeftTalon.get());
```

Comment:
we should remove these in favor of implementing Sendable. I know we did this to be simple and get it working, but now we need to do it the right way